ci-operator/config/open-cluster-management/insights-client/OWNERS

```diff
@@ -0,0 +1,20 @@
+approvers:
```

Comment:
The owners file will get synced with the upstream owners file. We should make these the same:
https://github.com/open-cluster-management/insights-client/blob/main/OWNERS#L5

Reply:
updated